integration_tests/tests/waf-regional.rs

```diff
@@ -0,0 +1,18 @@
+#![cfg(feature = "waf-regional")]
+
+extern crate rusoto_core;
+extern crate rusoto_waf_regional;
+
+use rusoto_waf_regional::{WAFRegional, WAFRegionalClient, ListRulesRequest};
+use rusoto_core::{DefaultCredentialsProvider, Region};
+use rusoto_core::default_tls_client;
+
+#[test]
+fn should_list_rules() {
+    let credentials = DefaultCredentialsProvider::new().unwrap();
+    let client = WAFRegionalClient::new(default_tls_client().unwrap(), credentials, Region::UsEast1);
+    let request = ListRulesRequest::default();
+
+    let result = client.list_rules(&request).unwrap();
```

Comment:
Good points.  I don't feel strongly about it and demonstrating how it behaves with expect vs unwrap makes a clear case for keeping the unwraps for this.

S3 tests are a bit intertwined since it creates a bucket and uses that for subsequent tests.  Having the integration test fail means we get to delete the bucket manually.  [We've discussed this issue before](https://github.com/rusoto/rusoto/issues/452) but haven't done anything about it yet.  It's also required for subsequent tests, so if we split the tests up and assume they run in a random order, each test will need to create and delete a bucket for its tests.

Refactoring the S3 integration tests would help the issue in my original comment - I'll make a ticket for that.